File: submissions/strcc2j5.cpp
#include <bits/stdc++.h>
#pragma GCC optimize "Ofast"
using namespace std;

int main() {
    cin.sync_with_stdio(0);
    cin.tie(0);
    int s,w,l,b,h;
    cin>>s>>w>>l>>b>>h;
    cout << "The area of the square is " << s*s << "\n";
    cout << "The area of the rectangle is " << w*l << "\n";
    cout << "The area of the triangle is " << b*h/2.0 << "\n";
    cout << "The sum of all areas is " << s*s+w*l+b*h/2.0 << "\n";
}

File: submissions/strcc2j5.py
# Square
square_side = int(input())
s=square_side ** 2

# Rectangle
rect_width = int(input())
rect_length = int(input())
r= rect_width * rect_length

# Triangle
tri_base = int(input())
tri_height = int(input())
t= tri_base * tri_height / 2

print("The area of the square is "+ str(s))
print("The area of the rectangle is "+ str(r))
print("The area of the triangle is "+str(t))
print("The sum of all areas is "+ str(s+r+t))